vpro-shared-util/src/main/java/nl/vpro/util/Copier.java

```diff
@@ -142,9 +143,30 @@ public void waitFor() throws InterruptedException {
     }
 
     public boolean isReady() {
```

Comment:
Thie method should always throw IOException, let the caller handle it if wanted so. You always want to propagate the error.